src/utils/diffTool.ts

```diff
@@ -0,0 +1,19 @@
+import * as vscode from "vscode";
+
+export async function setDiffCommand(config: vscode.WorkspaceConfiguration) {
+  const input = await vscode.window.showInputBox({
+    prompt: "Please enter your diff tool command (e.g. diff -rq).",
```

Comment:
Should change that to a visual command now - on Linux I used `meld` successfully, but the example should probably be however you are supposed to invoke Beyond Compare on MacOS, since that's the preferred tool of many (intended) users of assay.